system-addon/lib/TopStoriesFeed.jsm

```diff
@@ -23,11 +24,15 @@ const DOMAIN_AFFINITY_UPDATE_TIME = 24 * 60 * 60 * 1000; // 24 hours
 const STORIES_NOW_THRESHOLD = 24 * 60 * 60 * 1000; // 24 hours
 const SECTION_ID = "topstories";
 
+// The filename where stories and topics are stored locally
+const POCKET_DATA_FILE = "pocketData.json";
+
 this.TopStoriesFeed = class TopStoriesFeed {
   constructor() {
     this.spocsPerNewTabs = 0;
     this.newTabsSinceSpoc = 0;
     this.contentUpdateQueue = [];
+    this.pocketCache = new PersistentCache(POCKET_DATA_FILE);
```

Comment:
Looks like nothing else in the file explicitly referred to "pocket" before… I suppose it's supposed to be generic? @csadilek Maybe just `this.cache = new PersistentCache("topStories");` ?